autodoc.pl

```diff
@@ -1780,6 +1773,15 @@ ($fh, $section_name, $element_name, $docref)
     }
 
     chomp $pod;     # Make sure prints pod with a single trailing \n
+
+    # Replace this marker line in the pod with what we say it should expand
+    # to.
+    while ($pod =~ / ( __PLAIN_vs_NOCONTEXT_wording__ \( (\w+) \) ) /x) {
+        my $entire = $1;
+        my $this = $2;
+        $this = $PLAIN_vs_NOCONTEXT_wording =~ s/WHICH/$this/gr;
+        $pod =~ s/\Q$entire/$this/;
+    }
```

Comment:
Why loop and match (and re-match) manually?
```suggestion
    $pod =~ s{ \b __PLAIN_vs_NOCONTEXT_wording__ \( (\w+) \) }{
        my $this = $1;
        $PLAIN_vs_NOCONTEXT_wording =~ s/WHICH/$this/gr
    }xeg;
```

Reply:
Thanks, done